ts/Series/PackedBubble/PackedBubblePoint.ts

```diff
@@ -157,7 +157,7 @@ class PackedBubblePoint extends BubblePoint implements DragNodesPoint {
  *
  * */
 
-interface PackedBubblePoint extends NetworkgraphPoint {
+interface PackedBubblePoint extends Omit<NetworkgraphPoint, 'onMouseOver' | 'optionsToObject'> {
```

Comment:
Please keep it simple. If it doesn't match then it is not extending.

Reply:
Any tip on how I can avoid this 😬 ? Removing the omit-statement breaks the build, as does removing the extend-keyword entirely 🤔

Reply:
It is not compatible, so extends is misleading. Instead you can reference supported properties as type via `property: TheInterface['theproperty'];` in your declaration. Or you can explicitly omit a property by setting it to undefined, but that should not be done for functions, instead assign the `noop` function from globals in the implementation.

Reply:
turns out we did not need to mess with typing after all, there was some residue code which did not matter 😆